cni/log/logger_error.go

```diff
@@ -0,0 +1,12 @@
+package log
+
+type ErrorWithoutStackTrace struct {
+	Err error
+}
+
+func (se ErrorWithoutStackTrace) Error() string {
+	if se.Err == nil {
+		return ""
+	}
+	return se.Err.Error()
+}
```

Comment:
I think we should more closely follow the reference implementation that I provided. While this is an error implementation, I'm not sure how well it's going to work with `errors.Is`, and `errors.As`, since `(Error.WithoutStackTrace).Err` is entirely opaque to those functions. Embedding the `error` as done in the playground example and adjusting the formatting is more exactly what we want and a lighter-weight modification.